packages/account/src/Containers/Account/account.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
+import { FadeWrapper, Loading } from '@deriv/components';
+import { matchRoute, routes as shared_routes } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import PageOverlayWrapper from './page-overlay-wrapper';
+import { TRoute } from '../../Types';
+import 'Styles/account.scss';
+import { flatten } from 'Helpers/utils';
+
+type TAccountProps = RouteComponentProps & {
+    routes: Array<TRoute>;
+};
+
+/**
+ * Component that renders the account section
+ * @name Account
+ * @param history - history object passed from react-router-dom
+ * @param location - location object passed from react-router-dom
+ * @param routes - routes object passed from react-router-dom
+ * @returns React component
+ */
+const Account = observer(({ history, location, routes }: TAccountProps) => {
+    const { client, ui } = useStore();
+    const {
+        is_virtual,
+        is_logged_in,
+        is_logging_in,
+        is_pending_proof_of_ownership,
+        landing_company_shortcode,
+        should_allow_authentication,
+    } = client;
+    const { toggleAccountSettings, is_account_settings_visible } = ui;
+    // subroutes of a route is structured as an array of arrays
+    const subroutes = flatten(routes.map(i => i.subroutes));
+    let selected_content = subroutes.find(r => matchRoute(r, location.pathname));
+
+    React.useEffect(() => {
+        toggleAccountSettings(true);
+    }, [toggleAccountSettings]);
+
+    routes.forEach(menu_item => {
+        if (menu_item?.subroutes?.length) {
+            menu_item.subroutes.forEach(route => {
+                if (route.path === shared_routes.financial_assessment) {
+                    route.is_disabled = is_virtual;
+                }
+
+                if (route.path === shared_routes.trading_assessment) {
+                    route.is_disabled = is_virtual || landing_company_shortcode !== 'maltainvest';
+                }
+
+                if (route.path === shared_routes.proof_of_identity || route.path === shared_routes.proof_of_address) {
+                    route.is_disabled = !should_allow_authentication;
+                }
+
+                if (route.path === shared_routes.proof_of_ownership) {
+                    route.is_disabled = is_virtual || !is_pending_proof_of_ownership;
+                }
+            });
+        }
+    });
+
+    if (!selected_content) {
+        // fallback
+        selected_content = subroutes[0];
```

Comment:
is it being used anywhere?

Reply:
`selected_content` is just used as a flag in Line 64. Let me see if can be refactored